ios/samples/gltf-viewer/gltf-viewer.xcodeproj/project.pbxproj

```diff
@@ -565,6 +567,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				DEVELOPMENT_TEAM = F2VQ778RU8;
```

Comment:
Please remove `DEVELOPMENT_TEAM` changes to Xcode projects

Reply:
Thanks. It's removed.